compiler/integration-tests/test/fuzzy/fuzzy.test.ts

```diff
@@ -0,0 +1,106 @@
+import { expect } from "@esm-bundle/chai";
+import { initializeResolver } from "@noir-lang/source-resolver";
+import newCompiler, { compile } from "@noir-lang/noir_wasm";
+import { acvm, abi } from "@noir-lang/noir_js";
+import { getRandomMap, getRandomString, getRandomUint8Array } from "./helper";
+import * as TOML from "smol-toml";
+
+const { default: initACVM, executeCircuit, compressWitness } = acvm;
+const { default: newABICoder, abiEncode } = abi;
+
+const iterations = 5;
+
+describe("compile fuzz test", () => {
+  for (let i = 0; i < iterations; i++) {
+    it("should error compile() gracefully", async () => {
+      await newCompiler();
+      await newABICoder();
+      await initACVM();
+      const fuzzedNoirSource = getRandomString(
+        Math.floor(Math.random() * Math.pow(10, i)),
+      );
```

Comment:
I don't think that this is a particularly useful test. This input is going to look nothing like Noir source code so what's the benefit of running this test multiple times?

It would be better to run some of out `compile_failure` tests on the wasm as they're closer to meaningful failure cases.